pkgs/applications/video/kodi/default.nix

```diff
@@ -43,15 +43,15 @@ assert vdpauSupport -> libvdpau != null;
 assert useWayland -> wayland != null && wayland-protocols != null && waylandpp != null && libxkbcommon != null;
 
 let
-  kodiReleaseDate = "20191116";
-  kodiVersion = "18.5";
+  kodiReleaseDate = "20200301";
+  kodiVersion = "18.6";
   rel = "Leia";
 
   kodi_src = fetchFromGitHub {
     owner  = "xbmc";
     repo   = "xbmc";
-    rev    = "${kodiVersion}-${rel}";
-    sha256 = "0pcrraj1ddzrd296br10yjnaxgb3iym74xzixcakaqhhp00f5hf6";
+    rev    = "18.6-Leia";
```

Comment:
do not dry evaluate the variables here

Reply:
I fixed the line 53 and I guess i squashed the commits (not sure they were done right)

Reply:
Something went wrong - you are now touching 1300+ files.

Reply:
sorry for all the force pushing.... I redid the change all over to get rid of the old commits. Hope it is fine now... (no experience with squashing)